content/security/for-admins/hardened-desktop/settings-management/_index.md

```diff
@@ -11,9 +11,9 @@ aliases:
 >
 >Settings Management is available to Docker Business customers only.
 
-Settings Management is a feature that helps admins to control certain Docker Desktop settings on client machines within their organization.
+Settings Management is a feature that helps admins to control certain Docker Desktop settings on client machines within their organization. It is designed specifically for organizations who don’t give developers root access to their machines.
 
-With a few lines of JSON, admins can configure controls for Docker Desktop settings such as proxies and network settings. For an extra layer of security, admins can also use Settings Management to enable and lock in [Enhanced Container Isolation](../enhanced-container-isolation/index.md) which ensures that any configurations set with Settings Management cannot be modified by containers.
+Administrators can configure controls for Docker Desktop settings such as proxies and network settings. For an extra layer of security, admins can also use Settings Management to enable and lock in [Enhanced Container Isolation](../enhanced-container-isolation/index.md) which ensures that any configurations set with Settings Management cannot be modified by containers.
 
 It is available with [Docker Desktop 4.13.0 and later](/desktop/release-notes.md).
```

Comment:
Suggest - Should we also add another note regarding the 4.35 prerequisite for DD?